lms/static/scripts/frontend_apps/components/dashboard/AssignmentActivity.tsx

```diff
@@ -27,6 +27,7 @@ type StudentsTableRow = {
 export default function AssignmentActivity() {
   const { dashboard } = useConfig(['dashboard']);
   const { routes } = dashboard;
+  const { organizationPublicId } = useParams();
   const { assignmentId } = useParams<{ assignmentId: string }>();
```

Comment:
You can merge these two calls to `useParams()`

```suggestion
  const { assignmentId, organizationPublicId } = useParams<{ assignmentId: string; organizationPublicId?: string }>();
```